pkg/frontend/admin_openshiftcluster_kubernetesobjects_test.go

```diff
@@ -137,7 +165,7 @@ func TestAdminKubernetesObjectsGetAndDelete(t *testing.T) {
 			resourceID:   fmt.Sprintf("/subscriptions/%s/resourcegroups/resourceGroup/providers/Microsoft.RedHatOpenShift/openShiftClusters/resourceName", mockSubID),
 			objNamespace: "openshift-project",
 			objName:      "config",
-			mocks: func(tt *test, openshiftClusters *mock_database.MockOpenShiftClusters, kactions *mock_kubeactions.MockInterface) {
+			mocks: func(tt *test, a *mock_adminactions.MockInterface, oc *mockdatabase.MockOpenShiftClusters, s *mockdatabase.MockSubscriptions) {
```

Comment:
I shortened and standardized these names and standardized the order across the tests for readability.